templates/software_router/software-router-secret.yaml

```diff
@@ -0,0 +1,12 @@
+{{ if .Values.software_router.deploy }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "cytomine.name" . }}-software-router-secret
+type: Opaque
+data:
+  config.groovy: |-
+    {{ (tpl (.Files.Get "configs/software_router/groovy.conf") . ) | b64enc | nindent 4 }}
+  id_rsa: |-
+    {{ .Files.Get "configs/software_router/ssh_key" | b64enc | nindent 4}}
```

Comment:
Its is better to have a precreated secret with the rsa key pair, since it is not possible for helm to generate the publickey only the private one.